[2025-03-12T15:21:00.081+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-12T15:21:00.102+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_lake_pipeline.start_pipeline manual__2025-03-12T15:20:57.288293+00:00 [queued]>
[2025-03-12T15:21:00.112+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_lake_pipeline.start_pipeline manual__2025-03-12T15:20:57.288293+00:00 [queued]>
[2025-03-12T15:21:00.113+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-12T15:21:00.136+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): start_pipeline> on 2025-03-12 15:20:57.288293+00:00
[2025-03-12T15:21:00.143+0000] {standard_task_runner.py:72} INFO - Started process 260 to run task
[2025-03-12T15:21:00.153+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_lake_pipeline', 'start_pipeline', 'manual__2025-03-12T15:20:57.288293+00:00', '--job-id', '171', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline_dag.py', '--cfg-path', '/tmp/tmpry5a8oln']
[2025-03-12T15:21:00.160+0000] {standard_task_runner.py:105} INFO - Job 171: Subtask start_pipeline
[2025-03-12T15:21:00.200+0000] {logging_mixin.py:190} WARNING - /usr/local/lib/python3.10/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-03-12T15:21:00.266+0000] {task_command.py:467} INFO - Running <TaskInstance: data_lake_pipeline.start_pipeline manual__2025-03-12T15:20:57.288293+00:00 [running]> on host ***-scheduler
[2025-03-12T15:21:00.421+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_lake_pipeline' AIRFLOW_CTX_TASK_ID='start_pipeline' AIRFLOW_CTX_EXECUTION_DATE='2025-03-12T15:20:57.288293+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-12T15:20:57.288293+00:00'
[2025-03-12T15:21:00.424+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-12T15:21:00.443+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-03-12T15:21:00.444+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "Pipeline démarré"']
[2025-03-12T15:21:00.458+0000] {subprocess.py:99} INFO - Output:
[2025-03-12T15:21:00.461+0000] {subprocess.py:106} INFO - Pipeline démarré
[2025-03-12T15:21:00.461+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-03-12T15:21:00.498+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-12T15:21:00.499+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=data_lake_pipeline, task_id=start_pipeline, run_id=manual__2025-03-12T15:20:57.288293+00:00, execution_date=20250312T152057, start_date=20250312T152100, end_date=20250312T152100
[2025-03-12T15:21:00.575+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-12T15:21:00.587+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
